test/suites/node/ttl_test.go

```diff
@@ -0,0 +1,92 @@
+package nodettl_test
+
+import (
+	"testing"
+	"time"
+
+	"k8s.io/apimachinery/pkg/labels"
+	"knative.dev/pkg/ptr"
+
+	"github.com/aws/karpenter/pkg/apis/awsnodetemplate/v1alpha1"
+	"github.com/aws/karpenter/pkg/apis/provisioning/v1alpha5"
+	awsv1alpha1 "github.com/aws/karpenter/pkg/cloudprovider/aws/apis/v1alpha1"
+	"github.com/aws/karpenter/pkg/test"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+
+	"github.com/aws/karpenter/test/pkg/environment"
+)
+
+var env *environment.Environment
+
+func TestNodeTTL(t *testing.T) {
```

Comment:
Thoughts on adding to the utilization tests? I envision our scaleup/down/efficiency tests will live there.